doc/source/reference/c-api/coremath.rst

```diff
@@ -229,6 +249,66 @@ types as of Nov 2022), you need to restrict to C90-compatible syntax, e.g.:
 
         b = npy_log(a);
 
+A few utilities have also been added in
+``numpy/npy_math.h``, in order to retrieve or set the real or the imaginary
+part of a complex number:
+
+.. code-block:: c
+
+    npy_cdouble c;
+    npy_csetreal(&c, 1.0);
+    npy_csetimag(&c, 0.0);
+    printf("%d + %di\n", npy_creal(c), npy_cimag(c));
+
+.. versionchanged:: 2.0.0
+
+    The underlying C types for all of numpy's complex types have been changed to
+    use C99 complex types. Up until now the following was being used to represent
+    complex types:
+
+    .. code-block:: c
+
+        typedef struct { double real, imag; } npy_cdouble;
+        typedef struct { float real, imag; } npy_cfloat;
+        typedef struct {npy_longdouble real, imag;} npy_clongdouble;
+
+    Using the ``struct`` representation ensured that complex numbers could be used
+    on all platforms, even the ones without support for built-in complex types. It
+    also meant that a static library had to be shipped together with NumPy to
+    provide a C99 compatibility layer for downstream packages to use. In recent
+    years however, support for native complex types has been improved immensely,
+    with MSVC adding built-in support for the ``complex.h`` header in 2019. This
+    led to the decision to use native complex types, remove the compatibility layer
+    and stop shipping a static library which created a lot of build headaches for
+    downstream packages that wanted to use this.
```

Comment:
Have we already stopped shipping the library on HEAD?

Reply:
No, not yet. Should I remove this?

Reply:
I think the part after "header in 2019." can be removed for now.